geomstats/geometry/poincare_polydisk.py

```diff
@@ -9,10 +20,10 @@
 
 
 class PoincarePolydisk(ProductManifold):
```

Comment:
You have an old version of this file as well (there has been changes in the docstrings for example): rebase?